src/renderer/styles/base.css

```
@@ -41,7 +66,12 @@
   /* Type. One family, and a seven-step size scale. The sizes used to be
      eleven values picked per element, four of them on half-pixels, which is
      what made the app look assembled rather than designed. */
-  --font: "Malgun Gothic", "Segoe UI", system-ui, sans-serif;
+  /* The bundled file first, so every machine draws the same thing. The
+     installed copy and the system stack behind it are what a build with the
+     font stripped would fall back to. */
+  --font:
+    "Pretendard Variable", Pretendard, "Malgun Gothic", "Segoe UI", system-ui,
```

Stylelint (17.14.0)

[error] Expected "Pretendard" to be "pretendard" (value-keyword-case)

(value-keyword-case)